templates/course/favorite.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Template for favorite button.
+ *
+ * @author LifterLMS
+ * @package LifterLMS/Templates
+ *
+ * @since [version]
+ *
+ * @var int    $object_id   WP Post ID of the Lesson.
+ * @var string $object_type Type, 'Lesson'.
+ */
```

Comment:
```suggestion
/**
 * Template for favorite button.
 *
 * @author LifterLMS
 * @package LifterLMS/Templates
 *
 * @since [version]
 *
 * @param int    $object_id   WP Post ID of the object to mark/unmark as favorite.
 * @param string $object_type The object type. Currently only the 'lesson' type is possible.
 */
```